aws/etc/terraform/instance/bastion.tf

```diff
@@ -1,42 +1,31 @@
+resource "aws_instance" "bastion" {
+  availability_zone = "${data.aws_availability_zones.available.names[0]}"
```

Comment:
Did we determine that offest 0 is always the same?

Reply:
yes, afaict, although it still deletes the bastion and recreates it so it doesn't break anything if it were to change AZs